site/content/en/docs/technical-outcomes/scheduling/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Simulation Testing
+---
+
+# Simulation Testing with KWOK
+
+{{< hint "info" >}}
+
+This document walks you through the technical outcome of using KWOK for simulation testing.
+
+{{< /hint >}}
+
+This documentation describes the technical outcome of using KWOK for simulation testing in your environment.
+It goes over the key simulation features of using KWOK, and then provides a step-by-step guide to creating
+various scheduling scenarios using KWOK.
+
+## Key Simulation Features of KWOK
+
+1. **Lifecycle configuration:** The pod, node, and node lease can be simulated.
+
+    - **Pod:** Status information like Pending, ContainerCreating, Running, and Terminating can be simulated.
+    - **Node:** Status information like NodeReady, and Ready states can be simulated.
+
+2. **Kubelet server**
+
+    - **Metrics:** Using the [Kubernetes Metric Server](https://kwok.sigs.k8s.io/docs/examples/metrics-server/), or [Prometheus](https://kwok.sigs.k8s.io/docs/examples/prometheus/), metrics can be simulated in KWOK.
+    - **Exec:** Commands can be simulated to be [executed](https://kwok.sigs.k8s.io/docs/user/exec-configuration/) inside a container running on a single fake pod or multiple pods.
+    - **Log:** Container [logs](https://kwok.sigs.k8s.io/docs/user/logs-configuration/) can be simulated in a single pod or multiple pods.
+    - **Attach:** The input/output streams from the main process running in a container can be [attached](https://kwok.sigs.k8s.io/docs/user/attach-configuration/#attach-configuration) to KWOK.
+    This can be simulated in a single pod or multiple pods.
+    - **PortForward:** [Port forwarding](https://kwok.sigs.k8s.io/docs/user/port-forward-configuration/#clusterportforward), which establishes a direct connection to a container port, can be simulated
+    for a single pod or multiple pods.
```

Comment:
I think this part should be move to a new page `site/content/en/docs/technical-outcomes/_index.md`

Reply:
Ok, that also means that other technical outcomes may have to be referenced here too, correct?

Reply:
Since the aim is to have several pages introducing how to use KWOK combined with various scenarios, we can remove Lines 5 - 32.
This page focuses on scheduling not simulation.
There will be no need to move the page.

Reply:
Well, it's a wise choice, I also think it's a bit of a duplicate of the home page

Reply:
True, I will update it now.

Reply:
Also, we could update the title to "Scheduler Testing", since the scheduling process is real, using "Scheduling Simulation" as the title is confusing.

Reply:
What do you think?